pkg/internal/checkup/checkup.go

```diff
@@ -400,7 +415,6 @@ func newDPDKVMI(checkupConfig config.Config) *kvcorev1.VirtualMachineInstance {
 		vmi.WithHugePages(),
 		vmi.WithMemoryRequest("8Gi"),
 		vmi.WithTerminationGracePeriodSeconds(terminationGracePeriodSeconds),
-		vmi.WithNodeSelector(checkupConfig.DPDKNodeLabelSelector),
```

Comment:
Is it possible to now remove this function?

Reply:
DONE